_studio/shared/umc/codec/h264_dec/src/umc_h264_task_supplier.cpp

```diff
@@ -4053,7 +4055,8 @@ void TaskSupplier::DBPUpdate(H264DecoderFrame * pFrame, int32_t field)
 {
     H264DecoderFrameInfo *slicesInfo = pFrame->GetAU(field);
 
-    for (uint32_t i = 0; i < slicesInfo->GetSliceCount(); i++)
+    uint32_t count = slicesInfo->GetSliceCount();
+    for (uint32_t i = 0; i < count; i++)
```

Comment:
Why you don't use the same `for (auto pSlice : m_pSliceQueue) ` here as in the prev. patch?

Reply:
I think reason is that m_pSliceQueue is inaccessible from this layer

Reply:
@dvrogozh , yes @mgonchar  right.
m_pSliceQueue is protected member of H264DecoderFrameInfo

Reply:
ok